packages/react/src/helpers/validateShippingAddressZipCode.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  type AddressBase,
+  AddressType,
+  type CountryAddressSchema,
+} from '@farfetch/blackout-client';
+
+/**
+ * Zip Code Validation Errors
+ * * ER01 - The zip code validation is ignored because is not included in the countries array to be validated
+ * * ER02 - Zip code schema not found
+ * * ER03 - The zip code inserted is invalid
+ */
+export enum ZipCodeValidationError {
+  ZipCodeValidationCountryIgnored = 'ER01',
+  ZipCodeSchemaNotFoundError = 'ER02',
+  ZipCodeInvalid = 'ER03',
+}
+
+/**
+ * Validates zip code for an array of iso-codes and returns true if the address' country code is not in the array isoCodesToValidate,
```

Comment:
Adjust the jsdoc please as it does not match the implementation. Also add a note that if the country is not in the isoCodesToValidate, it will return false and the error code `ZipCodeValidationCountryIgnored` so that consumers can distinguish this case.

Reply:
pls double check